compute/gpu/how-to/create-manage-gpu-instance.mdx

```diff
@@ -0,0 +1,57 @@
+---
+meta:
+  title: How to create and manage a GPU Instance
+  description: This page explains how to create and manage a GPU Instance
+content:
+  h1: How to create and manage a GPU Instance
+  paragraph: This page explains how to create and manage a GPU Instance
+dates:
+  validation: 2021-12-14
+  posted: 2021-12-14
+---
+
+Scaleway GPU Instances are virtual compute instances equipped with dedicated high-end Nvidia graphical processing unit (GPUs). They are ideal for data processing, articifical intelligence, rendering and video encoding. After you have created your GPU Instance, you can connect to it via SSH and use one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) to access a preinstalled [Pipenv](/compute/gpu/how-to/use-gpu-with-docker) environment with all your favourite AI libraries and tools pre-installed. In addition to this, GPU Instances have all the features of our regular Instances, including [flexible IPs](/compute/instances/how-to/use-flexips/), [Security Groups](/compute/instances/how-to/use-security-groups/), [Private Networks](/compute/instances/how-to/use-private-networks/), [backups](/compute/instances/how-to/create-a-backup/) and more. When you've finished with your GPU Instance, you can easily delete it from the Scaleway Console.
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [created your SSH Key and added it to your account](/console/my-project/how-to/create-ssh-key/)
+
+</Message>
+
+## How to create a GPU Instance
+
+1. Click **Instances** in the **Compute** section of the side menu. The Instance creation page displays.
+
+2. Click **Create an Instance**. The Instance creation wizard displays.
+
+3. Complete the following steps in the wizard: 
+
+    - Choose an **Availability Zone**, which is the geographical region where your GPU Instance will be deployed.
+    - Select **GPU** as the **Instance type**, and then choose the type of GPU Instance you want. Different types have different prices, processing power, memory, SSD disks and bandwidth. 
+    - Choose an **Image** to run on your GPU Instance. [TODO CHECK WHAT THE IMAGE OPTIONS WILL LOOK LIKE]
```

Comment:
Don't forget to remove this part "[TODO CHECK WHAT THE IMAGE OPTIONS WILL LOOK LIKE]" after you have checked the image options :)